CONTRIBUTING.md

```diff
@@ -114,24 +114,47 @@ To release a new version of `datadog-ci`:
 1. Create a new branch for the version upgrade.
 2. Update the `package.json` version to `X.X.X`, commit the change `vX.X.X` and tag it with `git tag vX.X.X`.
    - You may refer to [Semantic Versioning](https://semver.org/#summary) to determine what level to increment.
-4. Push the branch **along with the tag** with `git push --tags origin name-of-the-branch`, create a PR, and get at least one approval.
+3. Push the branch **along with the tag** with `git push --tags origin name-of-the-branch`, create a PR, and get at least one approval.
    - **Find and open** the workflow run corresponding to your tag [in this list](https://github.com/DataDog/datadog-ci/actions/workflows/publish-release.yml).
    - Copy the release notes from the summary, and paste them in the description of your PR. This ensures the feature PRs have a link to your release PR.
    - Add the `release` label to your PR.
    - See this [example PR](https://github.com/DataDog/datadog-ci/pull/1215).
-5. Once you've received at least one approval, merge the PR **with the "Create a merge commit" strategy**.
+4. Once you've received at least one approval, merge the PR **with the "Create a merge commit" strategy**.
    - You may notice that a **GitHub** job is waiting for an approval, and some **_GitLab_** jobs are pending: this is expected (see **step 6 and 8**). You can merge the PR when *only those jobs* are left.
    - The "Create a merge commit" strategy is required for **step 7**, and for the GitHub Release to point to an existing commit once the PR is merged.
-6. The `npm-publish` job is waiting for an approval from a datadog-ci maintainer: ask for approval and wait for it and its downstream jobs to succeed.
-7. Go to the draft GitHub Release, and publish it as **latest**.
+5. The `npm-publish` job is waiting for an approval from a datadog-ci maintainer: ask for approval and wait for it and its downstream jobs to succeed.
+6. Go to the draft GitHub Release, and publish it as **latest**.
    - There should be 5 binaries available in the release's assets.
-8. Finally, go to the [_GitLab_ pipelines](https://gitlab.ddbuild.io/DataDog/datadog-ci/-/pipelines?scope=tags&status=manual), find the pipeline for your tag, and start the `build` stage to run the Docker image build jobs.
+7. Finally, go to the [_GitLab_ pipelines](https://gitlab.ddbuild.io/DataDog/datadog-ci/-/pipelines?scope=tags&status=manual), find the pipeline for your tag, and start the `build` stage to run the Docker image build jobs.
    - Make sure all the jobs and downstream jobs succeed.
 
 Thanks for creating a release! 🎉
 
 </details>
 
+##### Overwriting a release candidate
+
+Some PRs were just merged and you want to release them, but you already started the release process? 😬 Psst, come here!
+
+<details>
+  <summary>Instructions</summary>
+
+> [!WARNING]  
+> If you went through this process, please notify the reviewer so they can ensure everything is correct.
```

Comment:
This alert isn't rendering, I think because inside the `details` tag some Markdown isn't recognized:

<img width="908" alt="image" src="https://github.com/user-attachments/assets/4fd05e3e-4541-4feb-8a7b-a09d15471410" />